packages/botonic-react/tests/webchat/storage.test.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import TestRenderer, { act } from 'react-test-renderer'
+import { Webchat } from '../../src/webchat/webchat'
+
+describe('TEST: storage ', () => {
+  afterEach(() => {
+    localStorage.clear()
+    sessionStorage.clear()
+  })
+
+  it('Stores botonicState in the localStorage by default', async () => {
+    await act(async () => {
+      TestRenderer.create(<Webchat />)
+    })
+    expect(localStorage.getItem('botonicState')).not.toBeNull()
```

Comment:
maybe you can check here the contents of botonicState if it's not tested anywhere else?
if required, only check the contents in 1 test